ftdetect/bind-named.vim

```diff
@@ -11,4 +11,4 @@
 " Remarks:
 " See syntax/bind-named.vim for additional info.
 
-au! BufNewFile,BufRead named.conf,rndc.conf,rndc.key,*-named.conf,*_named.conf,*.named.conf,named-*.conf,named_*.conf,named.*.conf,rndc-*.conf,rndc_*.conf,rndc.*.conf,rndc.key,rndc-*.key,rndc_*.key,rndc.key,mz.*,sz.*,view.*,named.conf*  set filetype=bind-named
```

Comment:
*.zone* isn’t going to sit well with the public space of filename matching in the general Vim community.   It will catch a lot of files having .zone in the middle of the a filename.  The false positive would be too uncomfortable for this plugin to take as-is.  (risk alienating the end-users to removing this plugin.  

Can we do zone.* or *.zone

Reply:
You are right, it could break something, i didn't think about this. It worked for me, so i thought to give my changes back :slightly_smiling_face: 

I have .zone and .zones files, so i would suggest to add *.zone and *.zones, but it's up to you.